bootstrap.py

```diff
@@ -639,6 +639,26 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = "katello-capsule"
```

Comment:
the `katello-capsule` package doesn't exist on Katello servers, just Smart Proxies that have content (f.k.a Capsules)